client/my-sites/theme/hooks/use-bundle-settings.tsx

```diff
@@ -23,18 +26,10 @@ const WooOnPlansIcon = () => (
 	</svg>
 );
 
-/**
- * Hook to get the bundle settings for a given theme.
- * If the theme doesn't have a sotfware set defined, it returns `null`.
- */
-const useBundleSettings = ( themeId: string ): BundleSettings | null => {
-	const themeSoftwareSet = useSelector( ( state ) => getThemeSoftwareSet( state, themeId ) );
+const useBundleSettings = ( themeSoftware?: string ): BundleSettingsHookReturn => {
```

Comment:
It allows to get the settings by theme software or theme (through `useBundleSettingsByTheme`) now.